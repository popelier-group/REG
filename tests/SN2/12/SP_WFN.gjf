# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.75356500   -0.00005700    0.00020900
 H                 -0.86767000   -0.90616900   -0.56447400
 H                 -0.86767700    0.94204300   -0.50213100
 H                 -0.86761500   -0.03605800    1.06727900
 Br                 1.60753400    0.00000500   -0.00001600
 Cl                -3.29429800    0.00001500   -0.00005500

frame.wfn
