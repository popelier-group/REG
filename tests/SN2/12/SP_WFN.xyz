
6

C        0.008077    0.072205    0.636914
H        0.673162    0.915099    0.647669
H        0.401930   -0.902149    0.856990
H       -1.046491    0.242192    0.746148
Br       -0.022171   -0.193541   -1.708987
Cl        0.041113    0.357974    3.161309
