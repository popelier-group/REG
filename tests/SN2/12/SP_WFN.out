 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.12/Gau-138779.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.12/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe PID=    138781.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.75357  -0.00006   0.00021 
 H                    -0.86767  -0.90617  -0.56447 
 H                    -0.86768   0.94204  -0.50213 
 H                    -0.86762  -0.03606   1.06728 
 Br                    1.60753   0.00001  -0.00002 
 Cl                   -3.2943    0.00002  -0.00006 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.753565   -0.000057    0.000209
      2          1           0       -0.867670   -0.906169   -0.564474
      3          1           0       -0.867677    0.942043   -0.502131
      4          1           0       -0.867615   -0.036058    1.067279
      5         35           0        1.607534    0.000005   -0.000016
      6         17           0       -3.294298    0.000015   -0.000055
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.073744   0.000000
     3  H    1.073741   1.849263   0.000000
     4  H    1.073751   1.849246   1.849251   0.000000
     5  Br   2.361099   2.695626   2.695594   2.695697   0.000000
     6  Cl   2.540733   2.651087   2.651035   2.651281   4.901832
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.008077    0.072205    0.636914
      2          1           0        0.673162    0.915099    0.647669
      3          1           0        0.401930   -0.902149    0.856990
      4          1           0       -1.046491    0.242192    0.746148
      5         35           0       -0.022171   -0.193541   -1.708987
      6         17           0        0.041113    0.357974    3.161309
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         146.6354885           0.8504518           0.8504517
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       173.0650774143 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  7.35D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147513.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.38794722     A.U. after   11 cycles
            NFock= 11  Conv=0.96D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.63809-101.29813 -62.29105 -56.11023 -56.10809
 Alpha  occ. eigenvalues --  -56.10809 -10.08860  -9.21143  -8.49994  -6.97153
 Alpha  occ. eigenvalues --   -6.96873  -6.96873  -6.32738  -6.32039  -6.32039
 Alpha  occ. eigenvalues --   -2.43888  -2.43697  -2.43697  -2.43145  -2.43145
 Alpha  occ. eigenvalues --   -0.60257  -0.56123  -0.52412  -0.30592  -0.30591
 Alpha  occ. eigenvalues --   -0.18169  -0.12081  -0.12081  -0.09230  -0.09230
 Alpha  occ. eigenvalues --   -0.08986
 Alpha virt. eigenvalues --    0.09204   0.11413   0.14955   0.14955   0.15607
 Alpha virt. eigenvalues --    0.17866   0.18322   0.18322   0.20432   0.21766
 Alpha virt. eigenvalues --    0.21767   0.23445   0.23446   0.24755   0.30800
 Alpha virt. eigenvalues --    0.31707   0.31707   0.36108   0.39316   0.52618
 Alpha virt. eigenvalues --    0.54333   0.54333   0.56341   0.56341   0.61680
 Alpha virt. eigenvalues --    0.65196   0.65197   0.70142   0.75803   0.76294
 Alpha virt. eigenvalues --    0.76294   0.81247   0.81247   0.83784   0.83785
 Alpha virt. eigenvalues --    0.85635   0.90146   1.11009   1.11009   1.11335
 Alpha virt. eigenvalues --    1.11335   1.12483   1.15754   1.22502   1.22503
 Alpha virt. eigenvalues --    1.33767   1.53083   1.62733   1.62734   1.68972
 Alpha virt. eigenvalues --    1.97413   1.97415   1.97989   2.13559   2.13560
 Alpha virt. eigenvalues --    2.20169   2.20170   2.30302   2.49082   2.49084
 Alpha virt. eigenvalues --    2.57342   2.62162   2.62162   2.66920   2.76660
 Alpha virt. eigenvalues --    2.76660   2.99972   3.22304   3.85980   3.85981
 Alpha virt. eigenvalues --    4.32236   4.32236   4.35743   4.35743   4.55694
 Alpha virt. eigenvalues --    6.74687   7.73174   7.73175   7.82267   9.87181
 Alpha virt. eigenvalues --   24.04027  26.05354  26.05356  26.25797  48.04752
 Alpha virt. eigenvalues --  215.84572 289.93466 289.93466 290.024541020.81895
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.035614   0.417588   0.417593   0.417569   0.059004  -0.048686
     2  H    0.417588   0.516354  -0.032143  -0.032152  -0.040036  -0.011918
     3  H    0.417593  -0.032143   0.516351  -0.032149  -0.040042  -0.011921
     4  H    0.417569  -0.032152  -0.032149   0.516365  -0.040020  -0.011910
     5  Br   0.059004  -0.040036  -0.040042  -0.040020  35.603767   0.007002
     6  Cl  -0.048686  -0.011918  -0.011921  -0.011910   0.007002  17.775993
 Mulliken charges:
               1
     1  C   -0.298682
     2  H    0.182308
     3  H    0.182311
     4  H    0.182297
     5  Br  -0.549675
     6  Cl  -0.698560
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.248234
     5  Br  -0.549675
     6  Cl  -0.698560
 Electronic spatial extent (au):  <R**2>=           1143.5235
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -0.0746    Y=             -0.6460    Z=             -5.7062  Tot=              5.7432
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -43.7974   YY=            -44.3743   ZZ=            -89.4005
   XY=             -0.0672   XZ=             -0.5932   YZ=             -5.1648
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             15.3934   YY=             14.8164   ZZ=            -30.2098
   XY=             -0.0672   XZ=             -0.5932   YZ=             -5.1648
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             -1.6662  YYY=             -6.6943  ZZZ=           -165.1169  XYY=              0.7036
  XXY=             -1.5723  XXZ=            -17.9529  XZZ=             -1.6672  YZZ=            -14.6480
  YYZ=            -19.2847  XYZ=             -0.2780
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -70.7990 YYYY=            -88.2816 ZZZZ=          -1789.6984 XXXY=             -1.0723
 XXXZ=             -9.5786 YYYX=             -0.7863 YYYZ=            -78.2895 ZZZX=            -13.4230
 ZZZY=           -117.2179 XXYY=            -26.3836 XXZZ=           -251.6941 YYZZ=           -258.9106
 XXYZ=            -25.5097 YYXZ=             -2.3602 ZZXY=             -1.0134
 N-N= 1.730650774143D+02 E-N=-7.698481328241D+03  KE= 3.069963484185D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE773\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.753565,-0.000057,0.000209\H,0,-0.86767,-0.906169,-0.564474\H,0,
 -0.867677,0.942043,-0.502131\H,0,-0.867615,-0.036058,1.067279\Br,0,1.6
 07534,0.000005,-0.000016\Cl,0,-3.294298,0.000015,-0.000055\\Version=ES
 64L-G16RevC.01\State=1-A\HF=-3074.3879472\RMSD=9.569e-09\Dipole=2.2595
 348,-0.000056,0.0002065\Quadrupole=-22.9007956,11.4504475,11.4503481,0
 .0001831,-0.0006703,-0.0000061\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 My opinions may have changed, but not the fact that I am right.
                       -- Ashleigh Brilliant
 Job cpu time:       0 days  0 hours  0 minutes 16.5 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  9.3 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:03 2020.
